src/utils.ts

```diff
@@ -21,15 +21,8 @@ export function arraysAreIdentical<T>(arr1: T[], arr2: T[]): boolean {
   }
 
   // Sort both arrays
-  const sortedArr1 = [...arr1].sort();
-  const sortedArr2 = [...arr2].sort();
+  const sortedArr1 = [...arr1].sort().toString();
+  const sortedArr2 = [...arr2].sort().toString();
```

Comment:
@abdelfatah-ashour with the new comparison `[null, undefined]` will be equal to `[undefined, null]` since both will be converted to `,`.